crates/ef-testing/src/sequencer/setup.rs

```diff
@@ -0,0 +1,263 @@
+use blockifier::abi::abi_utils::{
+    get_erc20_balance_var_addresses, get_uint256_storage_var_addresses,
+};
+use blockifier::state::state_api::{State, StateResult};
+use reth_primitives::{Address, Bytes};
+use revm_primitives::U256;
+use starknet_api::core::Nonce;
+use starknet_api::hash::StarkFelt;
+use starknet_api::StarknetApiError;
+
+use super::constants::{
+    CONTRACT_ACCOUNT_CLASS_HASH, EOA_CLASS_HASH, FEE_TOKEN_ADDRESS, KAKAROT_ADDRESS,
+    PROXY_CLASS_HASH,
+};
+use super::types::FeltSequencer;
+use super::utils::{
+    class_hash_to_starkfelt, contract_address_to_starkfelt, get_storage_var_address,
+    split_bytecode_to_starkfelt, split_u256,
+};
+use super::KakarotSequencer;
+
+pub trait InitializeEvmState {
+    fn initialize_contract(
+        &mut self,
+        evm_address: &Address,
+        bytecode: &Bytes,
+        nonce: U256,
```

Comment:
is the nonce really a `U256`? I think that geth/reth use u64

Reply:
The nonce we get in the ef-tests is a `JsonU256`type. I can make the sequencer use U64 if you prefer.

Reply:
I am using u256 because this is the type used byt reth in the [ef-tests for the nonce](https://github.com/paradigmxyz/reth/blob/main/testing/ef-tests/src/models.rs#L208). I can make it u64, just means we will have a try_from somewhere else in the code.